decidim-elections/app/controllers/concerns/decidim/elections/orderable.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  module Elections
+    # Common logic to sorting resources
+    module Orderable
+      extend ActiveSupport::Concern
+
+      included do
+        include Decidim::Orderable
+
+        private
+
+        # Available orders based on enabled settings
+        def available_orders
+          @available_orders ||= begin
+            available_orders = %w(recent older)
+            available_orders
+          end
```

Comment:
It seems to me that it can be shortened to a line:

```suggestion
          @available_orders ||= %w(recent older)
```